src/lib/teams.js

```diff
@@ -0,0 +1,34 @@
+let teams = [
+  "Archifiltre",
+  "Code du travail numérique",
+  "DomiFa",
+  "e-MJPM",
+  "EgaPro",
+  "FCE",
+  "Maraudes",
+  "MedLé",
+  "Ops",
+  "Pass Préservatif / Tumeplay",
+  "RAMSES",
+  "Siao Data",
+  "TextStyle",
+  "Work In France"
+];
+
+export const sanitize = name =>
+  name
+    .replace(/[éè]/g, "e")
+    .replace(/[\s/;:,!?#]/g, "-")
+    .toLowerCase();
+
+teams = teams.reduce((teams, team) => {
+  teams.push({
+    name: team,
+    id: sanitize(team)
+  });
+  return teams;
+}, []);
```

Comment:
[map](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Objets_globaux/Array/map) is your friend 

also for sanitize we could use [some slugify package](https://github.com/sindresorhus/slugify)

Reply:
I should replace this `reduce` by a `map`... okay, gonna try it out.
I like our light `sanitize` :)